virttest/arch.py

```diff
@@ -89,4 +89,6 @@ def get_kvm_module_list():
         host_cpu_type = cpu.get_cpu_vendor_name()
         return ["kvm", "kvm-%s" % host_cpu_type]
     elif ARCH in ('ppc64', 'ppc64le'):
+        return ["kvm", "kvm-hv"]
+    elif ARCH in ('s390', 's390x'):
```

Comment:
Yes, this may a problem on POWER8, if the `kvm_hv` module is not loaded, `virttest/bootstrap.py` will report a warning.
BTW, `kvm_pr` has not parameters to set.
```bash
$ ls /sys/module/kvm_pr/
coresize  initsize   notes   rhelversion  srcversion  uevent
holders   initstate  refcnt  sections     taint
```

Reply:
> Yes, this may a problem on POWER8, if the `kvm_hv` module is not loaded, `virttest/bootstrap.py` will report a warning.
> BTW, `kvm_pr` has not parameters to set.
> 
> ```shell
> $ ls /sys/module/kvm_pr/
> coresize  initsize   notes   rhelversion  srcversion  uevent
> holders   initstate  refcnt  sections     taint
> ```

If you use the design II. taking a string like `kvm-hv:nested=1;kvm:` as input for `.reload`, could usage of `get_kvm_module_list` replaced by `kvm-pr:;kvm:param1=val1`?